proposals/0183-mobile-hash-managment.md

```diff
@@ -7,53 +7,71 @@
 
 ## Introduction
 
-Add automatic hash management to the manager layer of the libraries. Automatically send the resumption hash upon connection and refill the managers with the data they contained at the time that hash was retrieved.
+Add automatic hash management to the manager layer of the libraries. Automatically send the resumption hash upon connection and enable the reuse of data the developer sets if it already exists.
 
 ## Motivation
 
-With the addition of the manager layer, hashes have become significantly more difficult to use. While hashes can be sent and the data will resume on the system, the manager-layer does not know this and cannot restore the data that is currently in use. This may cause bugs or for data to be resent in its entirety that merely needs to be updated.
+With the addition of the manager layer, hashes have become significantly more difficult to use. While hashes can be sent and the data will resumed on the system, the manager-layer does not know this and cannot restore the data that is currently in use. This may cause bugs or for data to be resent in its entirety that merely needs to be reused.
```

Comment:
I think the second sentence needs to be updated: "...and the data will be resumed on..."